src/device/usbd.c

```diff
@@ -942,7 +943,7 @@ static bool process_get_descriptor(uint8_t rhport, tusb_control_request_t const
       // first byte of descriptor is its size
       return tud_control_xfer(rhport, p_request, (void*) desc_str, desc_str[0]);
     break;
-
+    }
```

Comment:
can you move the `}` before the `break;` , my style is having break outside of `}`. I admit not many people do it that way. But it is best to be consistent.

Reply:
It's also my style, just didn't pay attention :)